tests/playwright/scripts/version-util.cjs

```
@@ -0,0 +1,73 @@
+  if (!fs.existsSync(resolvedPath)) {
+    throw new Error(`Package.json not found: ${resolvedPath}`);
+  }
+  const fileContent = fs.readFileSync(resolvedPath, 'utf8');
+  const packageJson = JSON.parse(fileContent);
+  const originalVersion = packageJson.version;
+  const versionParts = parseVersion(originalVersion);
+  const [major, minor] = versionParts;
+  const newMinor = minor - 1;
+  let newVersion = `${major}.${newMinor}.0`;
+  if (newMinor < 0) {
+    console.log(`Minor cannot be less than 0, defaults to 1.20.0`);
```

Biome (2.1.2)

[error] Do not use template literals if interpolation and special-character handling are not needed.

Safe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)